src/css/style.css

```diff
@@ -36,54 +36,58 @@ body {
 .header {
   display: flex;
   justify-content: space-between;
-  padding: 0 6.25rem;
-  margin-top: 25px;
+  width: 90%;
+  margin: 0 auto;
+  padding-top: 45px;
 }
 .logo img {
-  width: 140px;
-  height: 120px;
+  width: 230px;
+  object-fit: cover;
 }
 .language {
+  height: 100%;
   display: flex;
+  gap: 15px;
   align-items: center;
 }
-.world {
-  display: flex;
-  align-items: center;
-  margin-right: 10px;
+.world img {
+  object-fit: contain;
 }
-.select-language {
-  display: flex;
-  align-items: center;
+.arrow {
+  appearance: none;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  background-image: url('../assets/Vector.png');
+  background-repeat: no-repeat;
+  background-position-y: center;
+  background-size: 12px;
 }
 .content-select select {
+  padding-right: 30px;
+  padding-left: 30px;
   width: 203px;
   height: 44px;
-  left: 1321px;
-  top: 68px;
-  text-align: center;
   font-size: 18px;
-  background: #ffffff;
+  background-color: #ffffff;
   border: 2px solid #c4c4c4;
-  box-sizing: border-box;
   border-radius: 10px;
-  padding-right: 30px;
-  padding-left: 30px;
-}
+  background-position-x: 93%;
+} 
 .input,
 .fieldset {
   width: 100%;
-  line-height: 16px;
-  margin-top: 0.5em;
-  padding: 1.1em 0.9em;
+  line-height: 65px;
```

Comment:
This is in reference to the height of the input. Should I use padding instead?
![Figma_5CKVd5mhQV](https://user-images.githubusercontent.com/71513679/174080246-13f0387f-f369-4fdc-bdc8-b2fd8c994672.png)

Reply:
65px line height I think is the default that have <h1> tag (not sure), just looks too much because it is in px, you could change it for:

`line-height: 4`

Too I think that Padding is easier to handle in this cases, but I don't have problem with use one or another

Reply:
done